### PR TITLE
Add support to the tutorial field in config.json

### PR DESCRIPTION
**Pull request by kaduartur**
**- What I did**
- I added support to the tutorial field in config.json
Example:
```json
[
  {
    "name": "sample_text",
    "type": "text",
    "label": "Type : ",
    "cache": {
      "active": true,
      "qty": 6,
      "newLabel": "Type new value. "
    },
    "tutorial": "Add a text for this field."
  },
  {
    "name": "sample_text",
    "type": "text",
    "label": "Type : ",
    "default": "test"
  },
  {
    "name": "sample_list",
    "type": "text",
    "default": "in1",
    "items": [
      "in_list1",
      "in_list2",
      "in_list3",
      "in_listN"
    ],
    "cache": {
      "active": true,
      "qty": 3,
      "newLabel": "Type new value?"
    },
    "label": "Pick your : ",
    "tutorial": "Select an item for this field."
  },
  {
    "name": "sample_bool",
    "type": "bool",
    "default": "false",
    "items": [
      "false",
      "true"
    ],
    "label": "Pick: ",
    "tutorial": "Select true or false for this field."
  },
  {
    "name": "sample_password",
    "type": "password",
    "label": "Pick: ",
    "tutorial": "Add a secret password for this field."
  }
]
``` 

**- How to verify it**
You can build this branch and add in the config.json the **tutorial** field.

**- Description for the changelog**
Support to the tutorial field in config.json

### CODE REVIEW

**Review by lucasdittrichzup — APPROVED** [review]

File: pkg/formula/runner/inputs.go [hunk 1]
```diff
@@ -135,16 +135,17 @@ func (in InputManager) fromPrompt(cmd *exec.Cmd, setup formula.Setup) error {
 				inputVal, err = in.loadInputValList(items, input)
 			} else {
 				validate := input.Default == ""
-				inputVal, err = in.Text(input.Label, validate)
+				inputVal, err = in.Text(input.Label, validate, input.Tutorial)
+
 				if inputVal == "" {
 					inputVal = input.Default
 				}
```
Comment (victor-schumacher) [inline pkg/formula/runner/inputs.go:142]:
``` go
if inputVal, err = in.Text(input.Label, validate, input.Tutorial); inputVal == "" {
    inputVal = input.Default
} 
```
what u think? :nerd_face:

**Review by victor-schumacher — APPROVED** [review]

**Comment by GuillaumeFalourd** [comment]
Great feature, thanks !

**Pull request merged**